common/src/main/java/com/wynntils/utils/render/Texture.java

```diff
@@ -86,6 +86,7 @@ public enum Texture {
     MAP_HELP_BUTTON("map/map_help_button.png", 10, 16),
     MAP_POINTERS("map/map_pointers.png", 256, 256),
     MAP_SHARE_BUTTON("map/map_share_button.png", 16, 14),
+    MAP_MANAGER_BUTTON("map/map_manager_button.png", 12, 16),
```

Comment:
Please keep these groups sorted.

Reply:
Is it meant to be alphabetical? Because if so these should be swapped too
`MAP_DEFENSE_FILTER_BUTTON("map/map_defense_filter_button.png", 16, 16),
     MAP_BUTTONS_BACKGROUND("map/map_buttons_background.png", 144, 26),`